docker-compose.yml

```diff
@@ -12,6 +12,9 @@ services:
     volumes:
       - ./docker-init-db.sql:/docker-entrypoint-initdb.d/init_db.sql
 
+  redis:
```

Comment:
Please add warning to CHANGELOG that redis is needed now

Reply:
And could you also add warning about using 2FA in the CHANGELOG (We use it now, right?)

Reply:
@joeribekker Do you think it's needed to add warnings? Afaik there hasn't been an official release of referentielijsten yet

Reply:
Adding a warning in the changelog can't hurt though

Reply:
added warnings for them